### PR TITLE
promote to production

### PR DESCRIPTION
**Pull request by ryancwalsh**
@sandoche Since Nick will soon be deploying `main`, I want to be sure that `main` is as updated as possible.

I know that I already have an open PR into `develop` (https://github.com/NEARFoundation/tx-tracking-app/pull/16), and if that one gets merged into `develop` before this one, that's great.

But if that one continues to take more time, maybe it makes sense to go ahead and merge the current `develop` into `main`.

### CODE REVIEW

File: server/src/services/getAccounts.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    const accounts: AccountStatus[] = [];
+    for (const accountId of accountIds) {
+      console.log('getAccounts', accountId);
+      const txTaskForAccount = await TxTasks.findOne({ accountId }).select({ __v: 0, _id: 0 });
+      const transactions = await TxActions.findOne({ accountId });
+      let lastUpdate = '';
+      let status = 'Pending';
+      if (txTaskForAccount) {
+        // if (txTaskForAccount.lastUpdate === 0) status = 'Pending';
+        if (txTaskForAccount.isRunning) status = 'In progress';
```
Comment (sandoche) [inline server/src/services/getAccounts.ts:23]: There are still if without brackets

File: shared/config.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    case 'production':
+    case 'mainnet':
+      return {
+        contractName,
+        exampleAccount: 'example.near',
+        explorerUrl: 'https://explorer.mainnet.near.org',
+        helperUrl: 'https://helper.mainnet.near.org',
+        networkId: 'mainnet',
+        nodeUrl: 'https://rpc.mainnet.near.org',
+        walletUrl: 'https://wallet.near.org',
```
Comment (sandoche) [inline shared/config.ts:18]: Note for the future, you will have to change it to mynearwallet or ideally to the wallet selector.

**Review by sandoche — APPROVED** [review]
Just added 2 comments and approved!

**Comment by ryancwalsh** [comment]
@sandoche  Thanks. I'll take note of those comments for a future commit to `develop` but will go ahead and merge this.

**Pull request merged**